### ISSUE TITLE
Update help section / review help section 

### ISSUE THREAD

**Issue by aashish24** [VIAME/VIAME-Web#191]

**Comment by BryonLewis** [comment]
I'm just going to use this issue as a storage mechanism for my ideas.

So the problem we are trying to solve is there there are several different keyboard shortcuts, modes and actions that can be done by a user with the annotation editor.  For this I am focusing on editing/creating and manipulating annotations.

Key features:
Quickly create, edit and label annotations in different modes
Detection mode - creating several annotations on a single frame of data
Track mode - creating tracks across frames of animation

**Detection Mode**
This mode is a little easier and we need to show how you can quickly create detections for each item in a single frame
1. Fastest way is to pick your default type, set to continuous and start drawing annotations
2. You can set to detection mode and utilize the Keyboard shortcut key 'N' to create a new detection

**Track Mode**
This mode is made for detecting or editing a track across several frames in an image sequence/video
1.  Advance frame will maintain the same track and advance to the next frame each time you create a detection
2.  Interpolation, allows you create keyframes and have the bounding box move between these keyframes

**Interpolation**
It will need it's own section for how to work with it.  The interpolation mode is complicated enough we should explain how it works.
**Basic interpolation:**  You draw a keyframe advance several frames forward and then position the editor to draw another frame.
**Interpolation controls:** the controls for advancing forward and back a keyframe through the interpolation
**Advanced interpolation:**  Creating breaks or occlusions in the track for the interpolation.

Type Editor:
Ability to toggle types on/off
Confidence slider for filtering data after being run through pipelines.
Change Type Name/Colors

TimeLine Viewers:
Event Viewer - showing track lengths and number of tracks
Timeline - showing number of each type of each time along a timeline view.

Keyboard Shortcuts:
Section dedicated to only keyboard shortcuts.

**Comment by BryonLewis** [comment]
For documentation purposes I'm working off of this document which will be transferred into the Wiki.  I am adding screenshots for everything.  I think once we have puppeteer installed for better integration testing we can utilize it's features to auto generate the screenshots we want of the UI for the help documentation.  Meaning we don't have to update screenshots everytime we create a new feature or edit the visual style of something we can automate that process.

[Help Doc in progress](https://docs.google.com/document/d/1WuMjszLl0w8WvVPRWGrXVu0ay7BALiPc5FOfYsCf20w/edit?usp=sharing)

**Comment by BryonLewis** [comment]
Okay I started adding the docs and figured out an easier way to add images and gifs to the wiki.  Now that we have an interface that is set for the interpolation and the new track settings I may need to re-record some of the gifs for the documentation.

The idea is to have some brief description of how to use certain tools and then have a short gif of the tool being used.

**Comment by BryonLewis** [comment]
[Updated Wiki](https://github.com/VIAME/VIAME-Web/wiki)

I've updated the wiki to restructure it and have it contain a lot more images/animations describing how to use specific functions and items.

There are still some small items that I want to update (The extensive interpolation animation).  I do believe that an eventual narrated video may help some people.

Something I talked earlier with Brandon about was the idea of using the wiki inside of the Annotation App to provide some direct contextual help.  That would probably be a future step.  The idea was to improve the existing documentation first so it is easier to go through.
